src/python/pants/backend/python/goals/run_pex_binary.py

```diff
@@ -72,9 +61,9 @@ async def create_pex_binary_run_request(
             interpreter_constraints=requirements_pex_request.interpreter_constraints,
             additional_args=field_set.generate_additional_args(pex_binary_defaults),
             internal_only=True,
-            # Note that the entry point file is not in the Pex itself, but on the
-            # PEX_PATH. This works fine!
-            entry_point=entry_point,
+            # Note that the entry point file is not in the Pex itself, but only present in the
+            # chroot. This works fine!
```

Comment:
This comment seems to have less information in it. The fact that it's in the chroot isn't sufficient. The fact that it's on the PEX_PATH, so that pex knows to load code from that location, is.

Reply:
Oh, did you mean `PYTHONPATH` originally? PEX_PATH is how you merge PEX files.